pkg/archive/jar.go

```diff
@@ -177,7 +183,7 @@ func splitPreservingQuotes(s string, sep rune) []string {
 	return result
 }
 
-var StableGitProperties = ZipEntryStabilizer{
+var StableJARGitProperties = ZipEntryStabilizer{
```

Comment:
in general, I'd prefer to have this in a separate PR since it's separate from the description of this one. if you'd like to leave it in now, I think that's fine but just some level-setting for the future.

Reply:
I thought about it too. I was perhaps just lazy to make another PR for this small change :sweat_smile: I will revert it and then send another PR for it :)